src/app/zones/sources.ts

```diff
@@ -22,7 +28,7 @@ export type ZoneEgressOverviewCollectionSource = DataSourceResponse<ZoneEgressOv
 
 export const sources = (api: KumaApi) => {
   return {
-    '/zone-cps': async (params: PaginationParams, source: { close: () => void }) => {
+    '/zone-cps': async (params: PaginationParams, source: Closeable) => {
```

Comment:
it's not strictly to this PR but seems like we're mixing up singular plural in our URLs, we have `.../mesh/...` but we have `.../policy/circuit-breakers/...` - do you know the reason for this? should we standardize on one? I think we could do what github does with `/pulls` for listing and `/pull/x` for single PR.

Reply:
> there, we have no control

we do have control on the back-end right? maybe we need to do a better job on the back-end

Reply:
Oh, sure. I was just talking about the frontend perspective. But then again, these are APIs consumed by other people and software, too. Harder to change than what’s isolated to the UI (the other two types).

Reply:
Just to note:

If I understand @slonka correctly here this is exactly what we should be doing for DataSource URIs

> I think we could do what github does with /pulls for listing and /pull/x for single PR.


i.e.

```
/zone-cps => list
/zone-cp/:name => item
```

Listings should use plurals, 'singles' should not.

Not super related to this but for browser URLs we should use plural if we are looking at a "summary view" i.e. listing plus detail, and single if we are looking at a detail view (which has no listing)

i.e. 

```
/mesh/default/services/service-1  => listing plus drawer (we don't have views like this yet)
/mesh/default/service/service-1 => just the detail view for service-1 i.e. no listing on the page
```

Reply:
In the backend we always use plural for resources regardless if it's a list or an item. Because Datasource separates things from the backend you can do whatever but it should at least be coherent inside the component